e2e/_suites/helm/helm_charts_test.go

```diff
@@ -12,8 +12,6 @@ import (
 	"testing"
 	"time"
 
-	"github.com/Jeffail/gabs/v2"
```

Comment:
I think those imports were removed by mistake

Reply:
My bad 😞 it's my editor that removes those for some reason.